README.md

```diff
@@ -14,27 +14,49 @@
 
 Cocotb can be installed by running `pip install cocotb`.
 
-## Quickstart
+## Usage
 
-    # Install pre-requisites (waveform viewer optional)
-    sudo yum install -y iverilog python-devel gtkwave
+For a D flip-flop description:
 
-    # Checkout git repositories
-    git clone https://github.com/cocotb/cocotb.git
+```verilog
+module dff (
+    output reg q,
+    input c, d,
+);
 
-    # Install cocotb
-    pip install ./cocotb
+always @(posedge clk) begin
+    q <= d;
+end
 
-    # Run the tests...
-    cd cocotb/examples/endian_swapper/tests
-    make
+endmodule
+```
 
-    # View the waveform
-    gtkwave waveform.vcd
+An example of simple randomized Cocotb based testbench:
 
+```python
+import random
+import cocotb
+from cocotb.clock import Clock
+from cocotb.triggers import FallingEdge
 
-## Tutorials and examples
+@cocotb.test()
+async def test_simple_dff(dut):
+    """ Test that D propagates to Q """
+    cocotb.fork(Clock(dut.c, 10, units="us").start())  # Create a 10us period clock on port `c`
```

Comment:
```suggestion
    
    clock = Clock(dut.c, 10, units="us") # Create a 10us period clock on port `c`
    cocotb.fork(clock.start()) # Start clock
```
Maybe?

Reply:
PEP-8 says to use double space before comments.